packages/smart-contracts/scripts/1_deploy-request-storage.ts

```diff
@@ -0,0 +1,45 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const addressBurnerContract = '0xfCb4393e7fAef06fAb01c00d67c1895545AfF3b8';
+
+// Deploys, set up the contracts
+export default async function deploy(_args: any, hre: HardhatRuntimeEnvironment) {
+  try {
+    const [deployer, recipient] = await hre.ethers.getSigners();
+    // Not used, only useful to keep the same addresses as with Truffle
+    const uselessInstance = await (await hre.ethers.getContractFactory('TestERC20')).deploy('100');
+    await uselessInstance.transfer(recipient.address, 10);
+
+    // Deploy the contract RequestHashStorage
+    const RequestHashStorage__factory = await hre.ethers.getContractFactory(
+      'RequestHashStorage',
+      deployer,
+    );
+    const RequestHashStorage = await RequestHashStorage__factory.deploy();
+    console.log('RequestHashStorage Contract deployed: ' + RequestHashStorage.address);
```

Comment:
seems this could use the `deployOne` utility

Reply:
Actually no because `deployOne` returns an address whereas here we need the contract instance later on.